README.md

```diff
@@ -23,7 +23,7 @@ If it is missing, you can install the latest stable release by running the follo
 $ kubectl create -f https://git.io/kube-dashboard
 ```
 
-If you are using Kubernetes 1.5 or earlier, you can install the latest stable release by running the following command:
+If you are using the Dashboard on cluster without authorization setup, you can install the latest stable release by running the following command:
```

Comment:
I'd also change `without authorization setup` to `without enabled authorization` or `with disabled authorization`.